test/image.bats

```diff
@@ -338,3 +338,18 @@ EOF
 	output=$(crictl images -o json | jq '.images[] | select(.repoTags[] == "quay.io/crio/hello-wasm:latest") | .pinned')
 	[ "$output" == "true" ]
 }
+
+@test "run container in pod with timezone set" {
```

Comment:
Not sure if we want to cover it here but it'd be nice to see what happens if time zone is invalid?

Not existent timezone and an empty string for CONTAINER_TIME_ZONE.

If for some reason we can't read the timezone database or it doesn't exist, do we panic or just ignore it?

Reply:
>Not sure if we want to cover it here but it'd be nice to see what happens if time zone is invalid?

I don't think we need an integration test for this. Runtime config validation will cover this use case, also addressed in `config_test.go`.

> Not existent timezone and an empty string for CONTAINER_TIME_ZONE.

For an empty string, it will not do anything, which means it will take the host machine time zone. This is as good as saying this feature doesn't exist.

>If for some reason we can't read the timezone database or it doesn't exist, do we panic or just ignore it?

We are not panicking but throwing appropriate errors wherever required, covered in `setContainerTimeZone`.

Reply:
Sounds good to me. As long as they are covered somewhere.